attributes/default.rb

```diff
@@ -0,0 +1 @@
+default['template_cookbook'] = 'automatic_updates'
```

Comment:
This attribute should probably be namespaced to something more like:
```
node['automatic_updates']['template_cookbook']
```

Just so we don't take `node['template_cookbook']` to conflict with every other cookbook that might use it.